packages/manager/apps/web/client/app/hosting/email/activate/activate.routing.js

```diff
@@ -0,0 +1,30 @@
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import sortBy from 'lodash/sortBy';
+import union from 'lodash/union';
+
+import {
+  WEB_HOSTING_NOT_USABLE_DOMAINS,
+} from './activate.constants';
+
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('app.hosting.activate', {
+    url: '/activate',
+    component: 'webHostingEmailActivateComponent',
+    resolve: {
+      serviceName: /* @ngInject */ $transition$ => $transition$.params().productId,
+      user: /* @ngInject */ User => User.getUser(),
+      domainNames: /* @ngInject */ ($q, Hosting, Domain) => $q.all({
+        domains: Domain.getDomains(),
+        hostings: Hosting.getHostings(),
+      })
+        .then(({ domains, hostings }) => sortBy(
+          filter(union(
+            domains,
+            hostings,
+          ), domain => !endsWith(domain, WEB_HOSTING_NOT_USABLE_DOMAINS)),
+        ))
+        .catch(() => []),
```

Comment:
This means you can't detect if the server is down for example, you should write an error message in the error component if this is the case